scylladb/scylla-rust-driver
# docs: update theme 1.3

Related issue https://github.com/scylladb/sphinx-scylladb-theme/issues/548

ScyllaDB Sphinx Theme 1.3 is now released 🥳 

We’ve added better redirects support and introduced numerous UI updates.

You can read more about all notable changes [here](https://sphinx-theme.scylladb.com/stable/upgrade/CHANGELOG.html#aug-2022).


## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

4. Open http://http://localhost:3000 with your favorite browser. The doc should render without errors, and the version should be Sphinx Theme version (see the footer) must be ``1.3.x``:

![image](https://user-images.githubusercontent.com/9107969/185590281-c79c5278-36d1-4bca-8197-e94824be1759.png)
